compiler/runtime/OMRAOTRelocationRuntime.hpp

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corp. and others
```

Comment:
2020, 2020

Reply:
Thank you for the comment, all kinds of mistakes like that are fixed within my contribution.